health_monitor/src/health_monitor.cpp

```diff
@@ -79,7 +79,8 @@ HealthMonitor::HealthMonitor(ros::NodeHandle &nodeHandle) : nodeHandle(nodeHandl
     diagnosticsUpdater.add(publisher_reportFault.add_check<diagnostic_tools::PeriodicMessageStatus>(
         "rate check", diagnostic_tools::PeriodicMessageStatusParams{}
                           .min_acceptable_period(minReportFaultRate)
-                          .max_acceptable_period(maxReportFaultRate)));
+                          .max_acceptable_period(maxReportFaultRate)
+                          .abnormal_diagnostic({diagnostic_tools::Diagnostic::ERROR, 1})));
```

Comment:
@patapufate is this change necessary?

Reply:
if we are gonna use diagnostics_tools with error:description msg in a future for diagnose health_monitor package, yes (from example, from mission_manager). if not, we could also remove all diagnostics from publisher_reportFault

Reply:
But do we have a fault defined for the `health_monitor`? We can generate a diagnostic w/o necessarily having a user facing fault for it.

Reply:
no. we don't have any fault definition for health monitor. I'll remove.